modules/pom.xml

```diff
@@ -23,6 +23,8 @@
         <aliases.xml.basedir>src/main/aliases</aliases.xml.basedir>
         <hibernate.search.version>${project.version}</hibernate.search.version>
         <hibernate.search.module.slot>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}</hibernate.search.module.slot>
+        <infinispan.integration.slot>for-hibernatesearch-${hibernate.search.module.slot}</infinispan.integration.slot>
```

Comment:
Sure, my question more is why HS engine itself declares that dependency instead of an app using that provider. It's not that you can "know" each possible provider. Is it just as a convincience for this specific one which you expect to be used frequently?